ext/node/polyfills/path/_constants.ts

```diff
@@ -2,6 +2,8 @@
 // Ported from https://github.com/browserify/path-browserify/
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
+// TODO: These constants are duplicated in ext/node/path/constants.rs
```

Comment:
Wanted to make sure we remembered to remove this file once the entire Path module is moved into Rust.. open to removing this comment.